src/components/pages/features/MetaproteomeAnalysisPage.vue

```diff
@@ -0,0 +1,378 @@
+                                    @filter-percentage-change="onUpdateFilterPercentage"
+                                />
+                            </v-window-item>
+                        </v-window>
+                    </v-card>
+                </v-col>
+            </v-row>
+        </template>
+
+        <v-row
+            v-else
+            class="search-info"
+        >
+            <v-col class="info-step">
+                <h2>Select samples</h2>
+                <p>
+                    Make a selection of samples that should be analysed. New samples can be added by pasting a peptide list, or existing samples can be reused.
+                </p>
+            </v-col>
+            <v-col class="info-step">
+                <h2>Click search</h2>
+                <p>
+                    Review the search settings for peptide deduplication, advanced missed cleavage handling and equating isoleucine and leucine before clicking search.
+                </p>
+            </v-col>
+            <v-col class="info-step">
+                <h2>Gain insight</h2>
+                <p>
+                    For each of the peptides, the lowest common ancestor (LCA) will be calculated, aggregated and visualised on the result page, giving you insight in the taxonomic diversity of your sample.
+                </p>
+            </v-col>
+        </v-row>
+    </div>
+</template>
+
+<script setup lang="ts">
+import LoadDatasetsCard from '@/components/cards/analysis/multi/LoadDatasetsCard.vue';
+import SelectDatasetCard from '@/components/cards/analysis/multi/SelectDatasetCard.vue';
+import SwitchDatasetCard from '@/components/cards/analysis/multi/SwitchDatasetCard.vue';
+import { onUnmounted, ref, watch } from "vue";
+import AnalysisSummaryCard from '@/components/cards/analysis/multi/AnalysisSummaryCard.vue';
+import {
+    GoSummaryCard,
+    EcSummaryCard,
+    InterproSummaryCard,
+    VisualizationOverview,
+    computeEcTree,
+    EcCode,
+    CsvUtils,
+    FunctionalSummaryProcessor,
+    PeptideCountTableProcessor,
+    useCsvDownload,
+    PeptideData,
+    FunctionalCode,
+    Ontology, NcbiTaxon, NcbiId, EcDefinition
+} from "unipept-web-components";
+import { storeToRefs } from 'pinia';
+import useMultiAnalysis from "@/store/MultiAnalysisStore";
+import SortingPeptidesDialog from "@/components/dialogs/SortingPeptidesDialog.vue";
+
+const multiAnalysisStore = useMultiAnalysis();
+
+const { activeAssayStatus } = storeToRefs(multiAnalysisStore);
+
+const selector = ref<boolean>(true);
+const displaySummary = ref<boolean>(false);
+const sortPeptidePercentage = ref<boolean>(false);
+
+const currentTab = ref<number>(0);
+
+const sortingPeptidesDialogOpen = ref<boolean>(false);
+
+const { downloadString } = useCsvDownload();
+
+watch(activeAssayStatus, () => {
+    console.log("Assay updated!");
+});
+
+watch(() => multiAnalysisStore.activeAssayStatus, () => {
+    console.log("Active assay updated!");
+});
+
```

Comment:
Put at the end of the file

Reply:
I have removed these functions since they contain redundant `console.log` invocations.